src/main/java/net/liquidpineapple/pang/XmlHandler.java

```diff
@@ -0,0 +1,270 @@
+package net.liquidpineapple.pang;
+
+import net.liquidpineapple.pang.gui.NumberToken;
+import net.liquidpineapple.pang.gui.ScoreSystem;
+import net.liquidpineapple.pang.gui.TimeSystem;
+import net.liquidpineapple.pang.logger.Logger;
+import net.liquidpineapple.pang.objects.Ball;
+import net.liquidpineapple.pang.objects.BallMovement;
+import net.liquidpineapple.pang.objects.GameObject;
+import net.liquidpineapple.pang.objects.Player;
+import net.liquidpineapple.pang.objects.playerschemes.Player1;
+import net.liquidpineapple.pang.objects.playerschemes.Player2;
+import net.liquidpineapple.pang.screens.LevelScreen;
+
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+import org.w3c.dom.Node;
+import org.w3c.dom.NodeList;
+import org.xml.sax.SAXException;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.ArrayList;
+
+import javax.imageio.ImageIO;
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+import javax.xml.transform.Transformer;
+import javax.xml.transform.TransformerException;
+import javax.xml.transform.TransformerFactory;
+import javax.xml.transform.dom.DOMSource;
+import javax.xml.transform.stream.StreamResult;
+
+/**
+ * Created by Jaap-Jan on 12-10-2016.
+ * Class which handles the parsing to and from XML files.
+ */
+public class XmlHandler {
+
+  /**
+   * Method which parses a Level into a xml file.
+   * @param level - Level to be parsed
+   * @param time - Time the level may take.
+   */
+  public static void createXmlFromLevel(LevelScreen level, int time) {
+    try {
+      DocumentBuilderFactory docFactory = DocumentBuilderFactory.newInstance();
+      DocumentBuilder docBuilder = docFactory.newDocumentBuilder();
+
+      // root elements
+      Document doc = docBuilder.newDocument();
+      Element rootElement = doc.createElement("level");
+      doc.appendChild(rootElement);
+
+      // background element
+      Element bg = doc.createElement("background");
+      bg.appendChild(doc.createTextNode("bg.png"));
+      rootElement.appendChild(bg);
+
+      //difficulty element
+      Element difficulty = doc.createElement("difficulty");
+      difficulty.appendChild(doc.createTextNode("1"));
+      rootElement.appendChild(difficulty);
+
+      // objects
+      Element objects = doc.createElement("objects");
+      rootElement.appendChild(objects);
+
+      // parse Balls
+      for (GameObject object : level.objectList) {
```

Comment:
Maybe this should be a seperate method to reduce the amount of lines in this method